### PR TITLE
#156185242 Setup app tests

### PR DESCRIPTION
**Pull request by leonardnjura**
#### What does this PR do?
- Manages dependency versions with variables defined in project build.gradle

#### Description of Task to be completed?
- Define dependencies in one place
- Refer to them using variable names

#### How should this be manually tested?
* Clone this repo: 
`$ git clone -b ft-testing-app-156185242 https://github.com/leonardnjura/github-users`
* Open cloned directory `github-users` in Android Studio
* Build with Gradle
* Connect an emulator or physical device
* Navigate to
`app/../test` and `app/../androidTest` 
* Run unit and instrumented tests using by right-clicking on IDE item or in a terminal using:
`$ ./gradlew test ` and `./gradlew connectedAndroidTest` 
* Expect passing tests
* To get a test report run
`./gradlew connectedAndroidTest`
* Open this Jacoco report in your browser after the tests
`app/build/reports/jacoco/jacocoTestReport/html/index.html`

#### What are the relevant pivotal tracker stories?
[#156185242](https://www.pivotaltracker.com/story/show/156185242)
[#157354404](https://www.pivotaltracker.com/story/show/157354404)
[#156185411](https://www.pivotaltracker.com/story/show/156185411)

#### Screenshots
ezgif com-video-to-gif (4)

ezgif com-video-to-gif (3)

Screenshot 2019-03-22 at 16 34 40

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (johngorithm) [inline .project:1]: gitignore this file

File: .settings/org.eclipse.buildship.core.prefs [hunk 2]
```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
```
Comment (johngorithm) [inline .settings/org.eclipse.buildship.core.prefs]: gitignore this file

File: app/.project [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (johngorithm) [inline app/.project:1]: gitignore this file

File: app/.settings/org.eclipse.buildship.core.prefs [hunk 4]
```diff
@@ -0,0 +1,2 @@
+connection.project.dir=
```
Comment (johngorithm) [inline app/.settings/org.eclipse.buildship.core.prefs]: gitignore this file

**Review by johngorithm — COMMENTED** [review]
Good Job 👍

File: .settings/org.eclipse.buildship.core.prefs (hunk 2 above)
Comment (leonardnjura) [reply .settings/org.eclipse.buildship.core.prefs]: Okay

**Review by johngorithm — APPROVED** [review]
LGTM.

**Comment by johngorithm** [comment]
I am merging this @leonardnjura

**Pull request merged**